webpack.config.js

```diff
@@ -3,7 +3,7 @@ const CircularDependencyPlugin = require('circular-dependency-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const path = require('path');
 
-const isProd = process.env.NODE_ENV === 'production';
+const isProd = false; // process.env.NODE_ENV === 'production';
```

Comment:
Let's not commit this - it might get in :)

Reply:
Should be fixed in newest commit